pkg/storagesvc/stowClient.go

```diff
@@ -48,8 +46,9 @@ type (
 )
 
 const (
-	StorageTypeLocal StorageType = "local"
-	PaginationSize   int         = 10
+	constStorageTypeLocal string = "local"
+	constStorageTypeS3    string = "s3"
+	PaginationSize        int    = 10
```

Comment:
- Constants need not pre-appended with `const` AFAIK as per idiomatic Go
- Are we making the storageTypeLocal now a package only variable?

Reply:
1. Done 
2. Yes. We have an interface level method which can be used from outside of package. It is `getStorageType() string`

Is this alright ?